bundler/lib/bundler/templates/newgem/test/minitest/test_newgem.rb.tt

```diff
@@ -2,7 +2,7 @@
 
 require "test_helper"
 
-class Test<%= config[:constant_name] %> < Minitest::Test
+class Test<%= config[:constant_name].gsub('::', '') %> < Minitest::Test
```

Comment:
:thinking: Is there any test covering this change?

Reply:
Good question, actually.

Reply:
Um, missing. I will add a test case for the test file containing "class TestFooBar < Minitest::Test" in the body.

Reply:
Add here 5f698fc

Reply:
It seems ok now @simi, go ahead and merge if it looks good to you!